content/blog/2021/08/DeployingCamelRouteInAWSLambda/index.md

```diff
@@ -0,0 +1,210 @@
+
+6. Once the function is created click the function name to upload the generated function.zip file and configure it. 
+
+7. Scroll down and select the Code tab. Click the upload from dropdown on right hand side of the screen and select .zip or .jar file 
+
+8. Click upload and browse to the path where the generated zip file is created `target/function.zip` and select the function.zip file and click save
+
+9. Under the Code tab scroll down to the Runtime settings and click edit 
+
+10. For the Handler details please provide the Quarkus Handler 
+
+```shell script
+io.quarkus.amazon.lambda.runtime.QuarkusStreamHandler::handleRequest
+```
+
+### Testing the AWS Lamda Function
+
+1. Now select the Test tab for executing a quick test. Copy paste the below json payload and hit Test 
+
+    ```shell script
+    {
+     "name": "Ravishankar"
+    }
+    ```
+2. If everything goes fine you should get the below response along with the lambda execution logs & stats
+
+    ```shell script
+    Hello Ravishankar ! How are you? from GreetService
+    ```
+## Deploying the Quarkus Native executable to AWS Lambda via AWS Web Console
+
+**_NOTE:_** [Please ensure that you have built your Java code as Quarkus Native executable](#native)
+
+1. Go to AWS Web console and search for Lambda Service
+ 
+2. Click Create Function and select Author From Scratch 
+
+3. Give the name for your function which should be unique
+
+4. For Runtime please scroll down and Select Provide your own bootstrap on Amazon Linux 2 under Custom Runtime 
+
+5. Under Permission feel free to create / use existing role to give the required permission for your lambda function
+
+6. Once the function is created click the function name to upload the generated function.zip file and configure it. 
+
+7. Scroll down and select the Code tab. Click the upload from dropdown on right hand side of the screen and select .zip or .jar file 
+
+8. Click upload and browse to the path where the generated a zip file is created `target/function.zip` and select the function.zip file and click save
+
+9. Under the Code tab scroll down to the Runtime settings and click edit 
+
+10. For the Handler details please provide the below Handler 
+
+    ```shell script
+        not.used.in.provided.runtime
+    ```
+
+11. Then Select the Configuration tab and click Environment Variables 
+
+12. For Key enter `DISABLE_SIGNAL_HANDLERS`	& for Value enter `true`
+
+### Testing the AWS Lamda Function
+
+1. Now select the Test tab for executing a quick test. Copy paste the below json payload and hit Test 
+
+    ```shell script
+    {
+     "name": "Ravishankar"
+    }
+    ```
+2. If everything goes fine you should get the below response along with the lambda execution logs & stats
+
+    ```shell script
+    Hello Ravishankar ! How are you? from GreetService
+    ```
+
+## JVM vs Native : Results based on lambda execution logs & stats
```

Comment:
As this is a blog, what do you think about sharing few comments about execution logs & stats like:
+ It's interesting to see that native uses less memory than JVM (maybe you can share the AWS cost saving if any?)
+ The execution time seems faster in native mode from 2nd to 5th invocations, it would be interesting to see if this trend continues on subsequent calls as the JIT compiler might kicks in
+ It's clear that the init duration is by magnitude faster in native. However, it's curious to see that the billed duration for the first call is higher in native mode, maybe a billing issue in AWS ?
+ Also could be please share a bit how the table was built ? Maybe it's given by AWS ? Or maybe you measured it on your own ?

Of course, feel free to add what you agree with. The key point is that I'm sure a lot of people would be interested about your feeling against this results. Do they confirm what you thought ? Are they some surprises ? and so on...